frontend/jupyter/src/styles.scss

```diff
@@ -4,7 +4,7 @@
 
 // Give some space after each input
 mat-form-field {
-  margin-bottom: 0.25rem;
+  margin-bottom: 0rem;
```

Comment:
Is there a reason for this change? upstream has this styling like it was.

Reply:
That is why we had a double line. I know upstream has it like that, my best guess it that the material ui has changed just a tiny bit.

Reply:
We've always had that double line, it's currently live on prod. I don't think this is the right idea to fix to fix something like that since this impacts the entire form page, even if it is very minor white space getting removed.
The double line issue seems like a separate issue based on the components we are using. Not on the generic kubeflow styling. 
Like on that topic, we also have a different background color than upstream for those input boxes in the dashboard page.
I think we should keep the same styling as upstream here, and make a separate dedicated ticket for fixing those filter input boxes, since this problem is not related to kf1.9.

Reply:
Reverted the change. Creating a follow-up ticket for this